schema_based_on_diagram.sql

```diff
@@ -0,0 +1,58 @@
+
+/* CREATE PATIENTS TABLE */
+CREATE TABLE patients (
+id int NOT NULL GENERATED ALWAYS AS IDENTITY,
+"name" varchar(100) NULL,
+date_of_birth date NULL,
+CONSTRAINT patients_pk PRIMARY KEY (id)
+); 
```

Comment:
- [ ] That should work however, let us change the following in this way
 ` "name" varchar(100) NULL, ` to ` name varchar(100) NULL,`

Reply:
Solved!